Modals/Fedosieiev Yehor - modals_task2.md

```diff
@@ -0,0 +1,51 @@
+#### Exercise 1. Choose the correct word or phrase. If both options are correct explain why.
+
+1. Hello. **Could** I speak to Mrs. Johnson, please?
+2. We **were allowed to** go home early yesterday because our techer was ill.
+3. The head teacher said we **could/were allowed to** go home.
+4. Do you think I **should** be worried about these spots on my forehead.
+5. You **ought to/should** enter that talent contest.
+6. You **shouldn't** talk to people like that. It's rude!
+7. Alan should **have written** two essays in the exam yesterday, not one.
+8. No, you **may** not go out tonight. You know you are grounded.
+9. Diana should have **waited/been waiting** for me at the corner. I wonder where she went.
+10. Yes, of course you **can** open the window if you're too hot!
+
+#### Exercise 2. Write a form of _must/have to/need to_ in each gap to complete the sentence.
+
+1. Oh, I must remember to get some potatoes on the way home tonight.
+2. Jason needs/has to see the headmaster during the next break. I wonder what's it about?
+3. We needed/had to light lots of candles during the power cut two nights ago.
+4. I'll have to start doing my Christmas cards soon. It's nearly December.
+5. Carl, you mustn't run into the street like that without looking first. It's dangerous.
+6. People with solar-powered cars don't have/need to worry about price of petrol.
+7. I wouldn't like to have to to get up at five o'clock every morning.
+8. We didn't need/have to do any washing-up after the picnic because we'd used disposable plates and cutlery.
+9. Do professional musicians need/have to practise every day?
```

Comment:
have to - as it is mostly their obligation but if it is necessity then need. So it is arguable